src/main/java/com/microsoft/bingads/internal/HttpClientWebServiceCaller.java

```diff
@@ -37,4 +45,41 @@ public HttpResponse post(URL requestUrl, List<NameValuePair> formValues) throws
 
     }
 
+    
+    private DefaultHttpClient createHttpClientWithProxy() {
+
+    	try {
+    		String webProxyEnabled = System.getProperty("web.proxy.enabled");
```

Comment:
yeah, I tried with ProxySeletor. When httpclient  post file, the schema of URI is "https". So If we use ProxySelector, the setting should be like the following;
       System.setProperty("https.proxyHost", "127.0.0.1");
        System.setProperty("https.proxyPort", "8888");
So I think ProxySelector is much better.